components/normalizer/Cargo.toml

```diff
@@ -46,7 +46,7 @@ databake = { version = "0.1.0", path = "../../utils/databake", optional = true,
 zerofrom = { version = "0.1.0", path = "../../utils/zerofrom" }
 
 [dev-dependencies]
-icu_testdata = { path = "../../provider/testdata" }
+icu_testdata = { path = "../../provider/testdata", default-features = false, features = ["icu_normalizer_serde"] }
```

Comment:
Suggestion: we've established that docs tests and integration tests all work fine with the baked provider. Therefore, rather than enabling `icu_normalizer_serde`, which implicitly enables this crate's "serde" feature, it is easy to simply `#[cfg(feature = "serde")]` all unit tests that require serde.

Clients of icu_testdata can and should enable the serde feature directly on their dependencies.